tests/ui/test_ui_tools.py

```diff
@@ -1983,6 +1983,7 @@ def test_main_view(self, mocker, message, last_message):
         [
             {
                 "id": 4,
+                "sender_id": 4,
```

Comment:
This does not appear to need this update?

I understand the desire to keep this message data up to date. We should likely be using something like a `message_fixture` from `conftest.py`. These tests were initially copy and pasted quite a lot, so a good refactor in a different commit/PR would be to simplify these by using our standard fixtures. One possible point to check is whether these are fixtures representing messages we receive from the server, and whether we update them later in the model (eg. modernize message response).